packages/grz-common/src/grz_common/utils/checksums.py

```diff
@@ -27,13 +27,7 @@ def calculate_sha256(file_path: str | PathLike, chunk_size=2**16, progress=True)
     sha256_hash = hashlib.sha256()
     with open(file_path, "rb") as f:
         if progress and (total_size > chunk_size):
-            with tqdm(
-                total=total_size,
-                unit="B",
-                unit_scale=True,
-                desc=f"Calculating SHA256 {file_path.name}",
-                smoothing=TQDM_SMOOTHING,
-            ) as pbar:
+            with tqdm(total=total_size, desc="SHA256  ", postfix=f"{file_path.name}", **TQDM_DEFAULTS) as pbar:  # type: ignore[call-overload]
```

Comment:
Pedantic, but isn't this `RAW` in `grz-check`?

Reply:
hm, you have a point, though grz-check does SHA256 + whatever else there is to do in one pass, so it's not a 1:1 mapping.
What I'm trying to say is: grz-cli validate will do a SHA256 pbar (and then either a FASTQ pbar or no additional pbar) while grz-check will only do a RAW pbar (which only does SHA256) _xor_ a FASTQ pbar (which does SHA256 + FASTQ)